src/os/ui/im/importmanager.js

```diff
@@ -30,31 +30,37 @@ os.ui.im.ImportManager = function() {
    * @type {Object<string, os.ui.im.IImportUI>}
    * @private
    */
-  this.importUIs_ = {};
+  this['importUIs_'] = {};
```

Comment:
Bracket notation should only be used for properties that we do not want the compiler to rename, which should never apply to private properties that are only directly referenced within the class. Please switch these back to dot notation, and if something needs to be accessed externally you can add an API call to do so.